README.md

```diff
@@ -475,7 +475,8 @@ okta:
       -----END PRIVATE KEY-----
     # or specify a path to a PEM file
     # privateKey: "/path/to/yourPrivateKey.pem" # PEM format. This SDK supports RSA AND EC algorithms - RS256, RS384, RS512, ES256, ES384, ES512.
-    requestTimeout: 0 # seconds
+    requestTimeout: 0                   # seconds
+    accessTokenTtl: PT1H                # duration in ISO-8601 format: PnDTnHnMn.nS
```

Comment:
Why can't we  simply stick to `seconds` here instead of a complicated format?

Reply:
Why not? :) 
It is not so complicated, but comfortable to use. 
Anyway, I've added support for both types - seconds also acceptable.